main.go

```diff
@@ -12,6 +15,24 @@ import (
 	"github.com/gorilla/handlers"
 )
 
+const segmentCDNHost string = "http://cdn.segment.com"
+const segmentTrackingAPIHost string = "http://api.segment.io"
+
+// Replace with real mirror host URLs
+var mirrorHosts = []string{
```

Comment:
You don't actually need to support multiple mirrors for our use case : we'll use an internal Kubernetes service to load balance between the pods serving this, and it will have a DNS name (see https://kubernetes.io/docs/concepts/services-networking/service/#dns)

Reply:
✓